app/views/annotation_categories/_annotation_text.html.erb

```diff
@@ -31,6 +31,13 @@
         <%= annotation_text.last_editor.user_name %>
       <% end %>
     </div>
+    <p>
```

Comment:
Okay this is probably the biggest change. I want you to move the logic for showing/hiding the table into the React component itself. I.e., move the button into the component, and register some state for either showing or hiding the table. This means you can put the "Annotation count: ___" text above or alongside the button, which is nice. (Could rename the button to "Show Details" or "Show Submissions".)

The thing to watch out for is that the data for the table should only be fetched when the table is first shown, which is now different than `componentDidMount`. This means that if the table is toggled repeatedly, the data won't be reloaded.